### PR TITLE
Events: reorder layers, add auto refresh

### PR DESCRIPTION
**Pull request by dmitry-vorobiev**
* event icons are rendered on top of the speed layer
* map will auto refresh itself if the expiration filter is set to 'Live'

### CODE REVIEW

File: src/events/EventsApp.js [hunk 1]
```diff
@@ -90,14 +91,46 @@ const EventsApp = observer(class App extends React.Component {
...
   }
 
+  static _fixLayerOrder(mapView) {
+    const layers = mapView.map.layers;
+    const iSpeed = layers.items.findIndex(lyr => lyr.id === 'speed');
+    if (iSpeed > 0) {
+      const speedLayer = layers.getItemAt(iSpeed);
+      layers.reorder(speedLayer, 0);
+    }
+  }
```
Comment (dmitry-vorobiev) [inline src/events/EventsApp.js]: I haven't found a better way to enforce layer order from the start. Updating layer settings in the config file didn't help.

Comment (ikaravan) [reply src/events/EventsApp.js]: how does it look in the ui?
I'm good with it if there is no impact

Comment (dmitry-vorobiev) [reply src/events/EventsApp.js]: image

Icons are rendered on top of the speed lines

File: src/events/EventsConfig.js [hunk 2]
```diff
@@ -164,15 +164,17 @@ const eventsConfig = {
...
     {
       id: 1, 
       type: "live", 
       name: "events1",
       popupTemplate: popupTemplate,
       showLegend: false,
-      outFields: ['eventType', 'eventTimestamp', 'eventExpirationTimestamp']
+      outFields: ['eventType', 'eventTimestamp', 'eventExpirationTimestamp'],
+      refreshInterval: 10
     },// baselineWhereCondition:eventsBaselineWhereCondition},
```
Comment (dmitry-vorobiev) [inline src/events/EventsConfig.js]: So now you can add `refreshInterval` in minutes if you want to enable auto-refresh. By default it's 0. 
I've deleted the whole 'live' stuff :)

**Pull request merged**